src/repository/entity/OwnerEntity.java

```diff
@@ -53,12 +53,20 @@ public String toString() {
                 + ", birthdate = " + super.getBirthDate() + ")";
     }
 
+
     @Override
     public int hashCode() {
 
-        return this.getName().length()
-                + this.getBirthDate().lengthOfMonth()
-                + this.getId().hashCode();
+        try {
+
```

Comment:
There is no need to add empty line here, becouse it is not method or class declaration